### ISSUE TITLE
Things that could need investigating (ongoing list)

### ISSUE THREAD

**Issue by cssprad1** [nasa-nccs-hpda/pytorch-caney#30]
This is an evolving list of configurations and settings that I have been using. I have not done more than verify these work as intended however they could stand to be looked over and see if it is best for our training needs

- Automatic resizing of data ([code](https://github.com/nasa-nccs-hpda/pytorch-caney/blob/main/pytorch_caney/data/transforms.py))
```python

        self.transform_img = \
            T.Compose([
                RandomResizedCropNP(scale=(0.67, 1.),
                                    ratio=(3. / 4., 4. / 3.)),
                T.ToTensor(),
                T.Resize((config.DATA.IMG_SIZE, config.DATA.IMG_SIZE)),
            ])
```

- Scaled learning rate ([code](https://github.com/nasa-nccs-hpda/pytorch-caney/blob/develop/pytorch_caney/lr_scheduler.py#L158))

```python

    linear_scaled_lr, linear_scaled_min_lr, linear_scaled_warmup_lr = \
        setup_scaled_lr(config)

```

- Choice of scheduler ([example](https://github.com/nasa-nccs-hpda/pytorch-caney/blob/main/examples/satvision/mim_pretrain_swinv2_satvision_base_192_window12_800ep.yaml))
```yaml
  LR_SCHEDULER:
    NAME: 'multistep'
    GAMMA: 0.1
```

- Best mask ratio ([example](https://github.com/nasa-nccs-hpda/pytorch-caney/blob/main/examples/satvision/mim_pretrain_swinv2_satvision_base_192_window12_800ep.yaml))
```yaml
  MASK_PATCH_SIZE: 32
  MASK_RATIO: 0.6
```

**Comment by cssprad1** [comment]
Done with #63
